packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -367,11 +367,11 @@ class DataGrid extends React.Component {
    * Keyboard Events
    */
   handleKeyDown(event) {
-    if (event.keyCode === KEYCODES.SHIFT) {
+    if (event.keyCode === KeyCode.KEY_SHIFT) {
```

Comment:
We should update [the import at the top of this file](https://github.com/cerner/terra-clinical/pull/479/files#diff-93badd98f0bc3febc86ceb7ac7123ad8L16) to use the keycode's module

```diff
- import { KEYCODES } from './utils/keycodes';
+ import KeyCode from 'keycode-js';
```

Then, we should be able to remove the `/utils/keycodes` file.

Reply:
This import is already there. https://github.com/cerner/terra-clinical/pull/479/files#diff-93badd98f0bc3febc86ceb7ac7123ad8R9